client/signup/steps/woocommerce-install/confirm/index.tsx

```diff
@@ -115,7 +117,7 @@ export default function Confirm( props: WooCommerceInstallProps ): ReactElement
 
 		return (
 			<>
-				<div className="confirm__info-section" />
+				<div className="confirm__info-section">{ isFetching && <LoadingEllipsis /> }</div>
```

Comment:
This shows for far longer than it should, we also don't disable the CTA while its loading, not a blocker for now imo but something that needs resolving. (nothing will come through here until we remove the woop flag checks)

I think its the requeststatusonce thing I created a while back (we call it in `useWooCommerceOnPlansEligibility` that's failing and retrying based on the error that comes back.

https://user-images.githubusercontent.com/811776/144368983-204f6013-76f6-414d-8b4d-82d365926fbd.mp4

Reply:
we've removed the code that was used to dispatch the actions to get the AT status. It isn't needed in the confirm step but the transfer one.